geocodio/data.py

```diff
@@ -171,32 +171,35 @@ def get(self, key):
         Returns an individual Location by query lookup, e.g. address, components dict, or point.
```

Comment:
I just realized looking it over, but as mentioned below, this `get` function "overrides" the default dict `get` function, so it should support the `default` parameter of the default dict get as function overloading isn't pythonic and not really supported. To support the `default` parameter, you might want to reorder the logic of lines 188-191 so that you first check if `key` is in `self.lookups`, and if so, set `key = self.lookups[key]` and then perform the `super.get(key, default)` with a propagated `default` parameter.

This will require additional `get` tests.

Reply:
Updated with a default value of `KeyError`, not sure if it's better to default to `None` as the parent dict `get` method does